crates/astria-conductor/src/execution_client.rs

```diff
@@ -23,12 +37,37 @@ pub(crate) trait ExecutionClient: crate::private::Sealed {
     async fn call_finalize_block(&mut self, block_hash: Vec<u8>) -> Result<()>;
 
     async fn call_init_state(&mut self) -> Result<InitStateResponse>;
+
+    // v1alpha2
```

Comment:
let's separate these all into a different `ExecutionClientV1Alpha2` trait (and rename this existing trait)